### PR TITLE
[OSDEV-862] Use the Test environment for creating anomymized dump

### PR DESCRIPTION
**Pull request by killev**
This PR fixes issue with lack storage on GitHub runner to contain 2+GB dump
* it fixes command in action
* it uses self-hosted runner to have more space

### CODE REVIEW

File: src/anon-tools/do_dump.sh [hunk 1]
```diff
@@ -18,6 +16,7 @@ chmod 600 /keys/key
 ssh -f -i /keys/key -L 5433:database.service.osh.internal:5432 -N ec2-user@$bastion
 
 pg_dump --clean --no-owner --no-privileges -Fc -h localhost  -d $DATABASE_NAME -U $DATABASE_USERNAME -p 5433 -f /dumps/osh_prod_large.dump -w --verbose
+ls -la /dumps
```
Comment (VadimKovalenkoSNF) [inline src/anon-tools/do_dump.sh:19]: Why we need to show everything in /dumps folder?

File: .github/workflows/db_apply_anonimized.yml [hunk 2]
```diff
@@ -13,8 +13,8 @@ on:
         default: Test
 
 jobs:
-  save-anonymized-db:
-    runs-on: ubuntu-latest
+  apply-anonymized-db:
+    runs-on: self-hosted
```
Comment (roman-stolar) [inline .github/workflows/db_apply_anonimized.yml:17]: Where do we hosted it?

**Review by roman-stolar — APPROVED** [review]
In general LGTM, just @VadimKovalenkoSNF & my questions.

File: .github/workflows/db_apply_anonimized.yml (hunk 2 above)
Comment (killev) [reply .github/workflows/db_apply_anonimized.yml:17]: This is my local build server

File: src/anon-tools/do_dump.sh (hunk 1 above)
Comment (killev) [reply src/anon-tools/do_dump.sh:19]: this is simple way to visually make sure and dump created correctly (simply by checking size)

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Review by VadimKovalenkoSNF — APPROVED** [review]
Approved. Pls, add in the ticket that it goes to release v1.12

**Pull request merged**